documentation/book/kafka-exporter/con-kafka-exporter-lag.adoc

```diff
@@ -29,5 +29,5 @@ Typical actions to reduce lag include:
 * Dropping messages
```

Comment:
What exactly do we mean with Dropping messages? I do not think that should be here.

Reply:
Dropping messages identified as non-critical according to a pattern or filter? Prefer to drop?

Reply:
There is no filter in Kafka. So I think we should drop this bullet point.

Reply:
removed